timeseries/src/autogluon/timeseries/__init__.py

```diff
@@ -21,4 +17,8 @@
         "`pip install mxnet==1.9` or `pip install mxnet-cu112==1.9` for GPU support."
```

Comment:
BTW, not related to this PR. But do we want to tell user to install `mxnet-cu112` when we don't know their cuda version?

Reply:
that's fair. updated.